apps/design/frontend/src/image_input.tsx

```diff
@@ -1,9 +1,11 @@
 import { Buffer } from 'node:buffer';
 import sanitizeHtml from 'sanitize-html';
-import { FileInputButton, FileInputButtonProps } from '@votingworks/ui';
-import { assert, assertDefined } from '@votingworks/basics';
+import { FileInputButton, FileInputButtonProps, P } from '@votingworks/ui';
+import { assert, assertDefined, err, ok, Result } from '@votingworks/basics';
+import { useEffect, useState } from 'react';
+import { safeParseNumber } from '@votingworks/types';
 
-const MAX_IMAGE_UPLOAD_BYTES = 5 * 1_000 * 1_000; // 5 MB
+const MAX_IMAGE_UPLOAD_BYTES = 1 * 1000 * 1_000; // 1 MB
```

Comment:
Why reducing to 1MB?

Reply:
Initially reduced to avoid unnecessarily large images in seal and signature fields, but doing the math on ballot measure images 5 MB may be more appropriate:

width = ~7 inches (accounting for ballot frame, padding etc) * 300 ppi = 2100 px
height = ~half a legal page max = 9 inches * 300 ppi = 2700 px

Not unreasonable for a 2100x2700 image to be up to 5 MB. Maybe in the future we want to add image size as a prop but not very high priority right now